mungegithub/mungers/submit-queue.go

```diff
@@ -51,11 +51,11 @@ const (
 	claYesLabel            = "cla: yes"
 	claHumanLabel          = "cla: human-approved"
 
-	jenkinsE2EContext  = "Jenkins GCE e2e"
-	jenkinsUnitContext = "Jenkins unit/integration"
-	jenkinsNodeContext = "Jenkins GCE Node e2e"
-	travisContext      = "continuous-integration/travis-ci/pr"
-	sqContext          = "Submit Queue"
+	jenkinsE2EContext    = "Jenkins GCE e2e"
+	jenkinsUnitContext   = "Jenkins unit/integration"
+	jenkinsVerifyContext = "Jenkins verification"
```

Comment:
Should they all be added to the stale ping checks? Currently jenkinsNodeContext is missing from that list.